ITISFoundation/osparc-simcore
# Adding SDS export format as FormatterV2

## What do these changes do?



Will add cmis standard exporting:

- [x] xlsx composing engine
- [x] formatter_V2 is now the SDS format
- [x] missing finish adding all fields to the SDS format when exporting
- [x] add V2 integration test project



## Related issue/s



https://github.com/ITISFoundation/osparc-issues/issues/425

## How to test



- rebuild and start `make build-devel-x up-devel`
- - make sure you have access to sleeper 2.1.1 (give yourself permissions in the database)
- Import the following project [project.zip](https://github.com/ITISFoundation/osparc-simcore/files/6203181/project.zip) (download and decompress) This helps you view the units in the inputs and outputs. In alternative create your own study
- go ahead an export or have a look at the new V2 format by replacing the extension  with `.zip` and extracting the archive

## Checklist
